### PR TITLE
enabling actions

### PR DESCRIPTION
**Pull request by ArturHarasimiuk**
Signed-off-by: Artur Harasimiuk <artur.harasimiuk@intel.com>

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
```
Comment (Panquesito7) [inline .github/workflows/ci.yml:5]:
```suggestion
on: [push]
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        compiler: [gcc-5, gcc-6, gcc-7, gcc-8]
+    env:
+      WORKFLOW: CI
+    steps:
+    - uses: actions/checkout@v2
```
Comment (Panquesito7) [inline .github/workflows/ci.yml:16]:
```suggestion
    - uses: actions/checkout@master
```

File: scripts/actions/run-action-build.sh [hunk 3]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -x
+#
+# Copyright (C) 2020 Intel Corporation
```
Comment (Panquesito7) [inline scripts/actions/run-action-build.sh:3]:
```suggestion
# Copyright (C) 2021 Intel Corporation
```

File: scripts/actions/run-action-docker-prepare.sh [hunk 4]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+# Copyright (C) 2020 Intel Corporation
```
Comment (Panquesito7) [inline scripts/actions/run-action-docker-prepare.sh:3]:
```suggestion
# Copyright (C) 2021 Intel Corporation
```

File: .github/workflows/verify.yml [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        compiler: [gcc-5, gcc-6, gcc-7, gcc-8]
+    env:
+      WORKFLOW: VERIFY
+    steps:
+    - uses: actions/checkout@v2
```
Comment (Panquesito7) [inline .github/workflows/verify.yml:13]:
```suggestion
    - uses: actions/checkout@master
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
I think you should also disable Travis CI, as GitHub Actions works much faster than Travis CI.

**Comment by JablonskiMateusz** [comment]
closing as deprecated

**Pull request closed without merging**